rules/windows/credential_access_generic_localdumps.toml

```diff
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2022/08/28"
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2022/08/28"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the enable of the full user-mode dumps feature system-wide. This feature allows Windows Error Reporting (WER)
+to collect data after an application crashes. This setting is a requirement for the LSASS Shtinkering attack, which
+fakes the communication of a crash on LSASS, generating a dump of the process memory, which gives the attacker access to
+the credentials present on the system without having to bring malware to the system. This setting is not enabled by
+default, and applications must create their registry subkeys to hold settings that enable them to collect dumps.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Full User-Mode Dumps Enabled System-Wide"
+references = [
+    "https://docs.microsoft.com/en-us/windows/win32/wer/collecting-user-mode-dumps",
+    "https://github.com/deepinstinct/Lsass-Shtinkering",
+    "https://media.defcon.org/DEF%20CON%2030/DEF%20CON%2030%20presentations/Asaf%20Gilboa%20-%20LSASS%20Shtinkering%20Abusing%20Windows%20Error%20Reporting%20to%20Dump%20LSASS.pdf"
+]
+risk_score = 73
+rule_id = "a7e7bfa3-088e-4f13-b29e-3986e0e756b8"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Credential Access"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+registry where registry.path : "HKLM\\SOFTWARE\\Microsoft\\Windows\\Windows Error Reporting\\LocalDumps\\DumpType"
```

Comment:
maybe exclude svchost services

`and not (process.executable : "?:\\Windows\\system32\\svchost.exe" and user.id : ("S-1-5-18", "S-1-5-19", "S-1-5-20"))`